### PR TITLE
Add systemd support

### PR DESCRIPTION
**Pull request by florian-zeidler**
@rebuy-de/prp-puppet-solr5  Add support for systemd. Because the service definitions are not automatically reloaded.

**Please review!**
- [x] Yes I added tests

### CODE REVIEW

**Comment by florian-zeidler** [comment]
@rebuy-de/prp-puppet-solr5 Please review.

File: manifests/service.pp [hunk 1]
```diff
@@ -3,6 +3,14 @@
   $solr_name
 ){
   if $manage_service {
+
+    if $::operatingsystem in ['RedHat', 'CentOS'] {
```
Comment (bjoernhaeuser) [inline manifests/service.pp]: you can use the fact service_provider from stdlib for that

Comment (svenwltr) [inline manifests/service.pp]: Shouldn't you also check the CentOS version?

Comment (florian-zeidler) [inline manifests/service.pp]: Yes and yes :D

**Comment by florian-zeidler** [comment]
@rebuy-de/prp-puppet-solr5 Better now?

**Comment by svenwltr** [comment]
:+1:

File: manifests/service.pp [hunk 2]
```diff
@@ -3,6 +3,14 @@
   $solr_name
 ){
   if $manage_service {
+
+    if $::service_provider == 'systemd'{
```
Comment (bjoernhaeuser) [inline manifests/service.pp]: if you add space before the `{` :)

**Comment by bjoernhaeuser** [comment]
:+1:

**Pull request merged**